lib/models/spons_category.dart

```diff
@@ -0,0 +1,9 @@
+/// Essential Model to process and get categories to process Response.
```

Comment:
This model is not required.
One variable classes are never suggestive. If you have to save just one variable, store it normally.